curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -628,3 +628,494 @@ Which of the following is the default value of the `position` property?
 
 `static`
 
+## --quiz--
+
+### --question--
+
+#### --text--
+
+Which position value positions the element according to the normal flow, then offsets it without affecting other elements?
+
+#### --distractors--
+
+`sticky`
+
+---
+
+`absolute`
+
+---
+
+`static`
+
+#### --answer--
+
+`relative`
+
+### --question--
+
+#### --text--
+
+Which position value positions the element relative to the nearest positioned ancestor?
+
+#### --distractors--
+
+`static`
+
+---
+
+`relative`
+
+---
+
+`sticky`
+
+#### --answer--
+
+`absolute`
+
+### --question--
+
+#### --text--
+
+When you apply top: 50% to an absolutely positioned element, what does that 50% refer to?
+
+#### --distractors--
+
+`50% of the element’s own height`
+
+---
+
+`50% of the viewport height`
+
+---
+
+`50% of the nearest ancestor’s width`
+
+#### --answer--
+
+`50% of the nearest positioned ancestor’s height`
+
+### --question--
+
+#### --text--
+
+If you set position: absolute on an inline element (like a <span>), what does its display type become according to the spec?
+
+#### --distractors--
+
+`It remains inline by default`
+
+---
+
+`It becomes display: table`
+
+---
+
+`It becomes a flex container`
+
+#### --answer--
+
+`It’s computed as a block‐level box`
+
+### --question--
+
+#### --text--
+
+How does overflow: hidden on a parent affect a child with position: sticky?
+
+#### --distractors--
+
+`It disables sticky behavior entirely`
+
+---
+
+`It makes the child scroll with the page normally`
+
+---
+
+`It forces the child to act like position: fixed`
+
+#### --answer--
+
+`It confines the sticky behavior so the child only sticks within the parent’s bounds` 
+
+### --question--
+
+#### --text--
+
+Which CSS property—alone, without any z-index—can create a new stacking context?
+
+#### --distractors--
+
+`margin`
+
+---
+
+`border`
+
+---
+
+`position: static`
+
+#### --answer--
+
+`transform` 
+
+### --question--
+
+#### --text--
+
+Which ancestor does an absolutely positioned element use as its containing block?
+
+#### --distractors--
+
+`The root <html> element`
+
+---
+
+`The viewport`
+
+---
+
+`The first element with display: block`
+
+#### --answer--
+
+`The nearest ancestor whose position is not static` 
+
+### --question--
+
+#### --text--
+
+Which of these is ignored by position: fixed when determining its containing block?
+
+#### --distractors--
+
+`The <body> element`
+
+---
+
+`A parent with position: relative`
+
+---
+
+`The root element`
+
+#### --answer--
+
+`All ancestors—position: fixed elements are always relative to the viewport` 
+
+### --question--
+
+#### --text--
+
+If two positioned elements overlap without any z-index specified, what determines which one paints on top?
+
+#### --distractors--
+
+`The element with larger width`
+
+---
+
+`Alphabetical order of their class names`
+
+---
+
+`Their position value (absolute over relative)`
+
+#### --answer--
+
+`Their order in the HTML (later elements paint on top)` 
```

Comment:
these don't need to be wrapped in backticks. please fix.
plus they need periods.